apps/examples/react-ssr-example/src/client/index.tsx

```diff
@@ -33,12 +33,14 @@ function AppWrapper() {
  * When using hot module replacement we need to use the render method
  * otherwise errors may occur in development.
  */
-const renderMethod = module.hot ? ReactDOM.render : ReactDOM.hydrate;
+/* tslint:disable no-string-literal */
+const renderMethod = module["hot"] ? ReactDOM.render : ReactDOM.hydrate;
```

Comment:
Not sure why bit for some reason the types for these things just stopped working. Tried to debug but eventually gave up and just did this instead.